test/score.test.js

```diff
@@ -0,0 +1,38 @@
+const expect = require('chai').expect;
+const { analyzeSentimentOfText, analyzeSentimentAndSaveScore } = require('../src/sentiment.js');
+
+/* eslint-disable no-undef */
+describe('analyzeSentimentOfText', () => {
+  /* eslint-disable no-undef */
```

Comment:
@meghanprestemon I think that if you move this `eslint-disable` comment up to the top of the file, it can apply to the entire file so that you will not need to repeat it so many times throughout the tests.